### DISCUSSION TITLE
Foldseek database download error

### DISCUSSION THREAD

**Discussion by zkskek321** [facebookresearch/esm#400] [ESM Metagenomic Atlas]
Hi, nice meet you again.

I'm trying to download foldseek search databsae using code described below link.
(https://github.com/facebookresearch/esm/blob/main/scripts/atlas/README.md)

Here is code -> aria2c --dir=/path/to/download/to --input-file=v0/highquality_clust30/foldseekdb.txt

But, there is one error with file https://dl.fbaipublicfiles.com/esmatlas/v0/highquality_clust30/foldseekdb/highquality_clust30_ss.index2.fastq.
I cannot download this file.
Except this file, I could download all another foldseek database files.

And I tried to download this file with "wget", but It shows error 'ERROR 403: Forbidden'.

Please check this error.

Thank you very much.

**Comment by tomsercu** [comment]
The `.txt` file doesn't contain the `2.fastq` part, looks like it got corrupted on local download. https://github.com/facebookresearch/esm/blob/main/scripts/atlas/v0/highquality_clust30/foldseekdb.txt

**Reply by zkskek321** [reply]
Oh it was my mistake. Thank you for your kindness!
